### DISCUSSION TITLE
12 months on one page and horizontal.

### DISCUSSION THREAD

**Discussion by Musteri1** [demark-pro/react-booking-calendar#9] [Q&A]
love your calendar. I want a horizontal calendar and also 12 months on one page.
but this calendar has a "downward" scrolling example, I can't scroll sideways because the days are at the top of the page.
I couldn't figure these out I can't import them to the page.
I would be very happy if you could create a sample page with these for me.
CalendarContainer
```
<CalendarContainer>
  <MonthContainer>
    <MonthArrowBack />
    <MonthContent />
    <MonthArrowNext />
  </MonthContainer>

  <WeekContainer>
    <WeekContent />
  </WeekContainer>

  <DaysContainer>
    <DayContainer>
      <DayContent />
      <DayToday />
      <DaySelection />
      <DayReservation />
    </DayContainer>
  </DaysContainer>
</CalendarContainer>
```

I'm a bit of a newbie in React, sorry for my question.
Best Regards

**Comment by demark-pro** [comment]
Hello. That's a good idea, I think I'll create a separate component for the horizontal calendar in the next version. Thank you for your support
